.github/workflows/deploy.yml

```diff
@@ -15,10 +15,9 @@ jobs:
       - name: Install and Build 🔧 # This example project is built using npm and outputs the result to the 'build' folder. Replace with the commands required to build your project, or remove this step entirely if your site is pre-built.
         run: make
       - name: Prepare Deploy folder
-        run: mkdir deploy && rsync -av --exclude=.git --exclude=.gitignore --exclude=deploy --exclude=.gitignore . deploy/
+        run: mkdir deploy && rsync -av --exclude=.git --exclude=.gitignore --exclude=deploy . deploy/
       - name: Deploy 🚀
-        uses: JamesIves/github-pages-deploy-action@3.5.7
+        uses: JamesIves/github-pages-deploy-action@4.0.0
         with:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this unnecessary because it's available by default?

Reply:
Ah, yes. Actually these tokens are handled in background (by github itself), since this kind of parameters were pointed as possible source of information leak.